bigquery_etl/cli/query.py

```diff
@@ -862,6 +877,13 @@ def _run_query(
         if not use_public_table and destination_table is not None:
             # destination table was parsed by argparse, however if it wasn't modified to
             # point to a public table it needs to be passed as parameter for the query
+
+            if re.match(QUALIFIED_TABLE_NAME_RE, destination_table):
+                project, dataset, table = qualified_table_name_matching(
+                    destination_table
+                )
+                destination_table = "{}:{}.{}".format(project, dataset, table)
```

Comment:
```suggestion
                destination_table = "{}.{}.{}".format(project, dataset, table)
```

usually we use the `project.dataset.table` notation for fully qualified tables

Reply:
To pass `destination_table` into `query_arguments`, I think it needs the `{}:{}.{}` notation.  See https://github.com/mozilla/bigquery-etl/blob/28b06a9dd45fc76b5e12687b264da21de0f571df/bigquery_etl/cli/query.py#L857

Reply:
A voice in the distance whispers: "we should migrate to the bigquery API..." 😄